.gitignore

```diff
@@ -1,3 +1,5 @@
+/data/output
```

Comment:
The `data/output` directory will likely hold two sub-directories:

* `scan`, which will have the latest scan data from `domain-scan` in it (`inspect.csv`, `tls.csv`, etc.).
* `processed`, which will have the output of the `data/update` script.